modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFormInstanceRecordLocalServiceImpl.java

```diff
@@ -923,11 +926,14 @@ protected void updateDDMContent(
 	}
 
 	protected void updateFormInstanceRecordVersion(
-		User user, DDMFormInstanceRecordVersion ddmFormInstanceRecordVersion,
-		int status, String version, ServiceContext serviceContext) {
+		DDMFormInstanceRecordVersion ddmFormInstanceRecordVersion,
+		String formInstanceVersion, int status, User user, String version,
+		ServiceContext serviceContext) {
```

Comment:
@rodrigopaulino why did you decide to change the order, putting `user` after `status`, but you kept `serviceContext` after `version`? 
I think we should follow the same pattern as other `LocalServiceImpls` (like [here](https://github.com/liferay/liferay-portal/blob/7eeb47e153c770522d597ed27d0dc975c14e35b1/modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMStructureLocalServiceImpl.java#L1561-L1594)) and keep the `User user` at the beginning.

Reply:
I'm just confused when it comes to these standards. 😑
Thanks for pointing it out, @carolmariaabb.
I'll change it now.